pipeline/article_processing/controller.py

```diff
@@ -0,0 +1,196 @@
+from __future__ import division
+import itertools
+import pandas as pd
+import string
+from nltk.corpus import stopwords
+from nltk.tokenize import RegexpTokenizer
+from nltk.stem import SnowballStemmer
+from nltk import pos_tag
+from datetime import datetime
+
+
+# Manual invalid title and link
+maintenance_title = ['Reduced service at Agrimoney.com',
```

Comment:
These operations would be easier to include in the scraper... I'm working on that right now.